tests/page-objects/reports/reports.po.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
   firstReport: () => element(by.css(`${reportListID} li:first-child`)),
   listLoader: () => element(by.css(`${reportListID} .loader`)),
   list: () => element(by.css(reportListID)),
-  reportByUUID: uuid => module.exports.list().all(by.css('li[data-record-id="' + uuid + '"]')),
+  reportByUUID: uuid => module.exports.list().all(by.css('li[data-record-id="' + uuid + '"]')).first(),
+  reportsByUUID: uuid => module.exports.list().all(by.css('li[data-record-id="' + uuid + '"]')),
```

Comment:
Some template string goodness to tidy that up and make sure we don't end up in quoting trouble, and then extracting to a variable to be sure we're conveying that they're the same thing:

```suggestion
  const reportUUIDSelector = `li[data-record-id="${uuid}"]`;
  reportByUUID: uuid => module.exports.list().all(by.css(reportUUIDSelector)).first(),
  reportsByUUID: uuid => module.exports.list().all(by.css(reportUUIDSelector)),
```